bfdriver/src/platform/efi/gen_modules.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Bareflank Hypervisor
+# Copyright (C) 2018 Assured Information Security, Inc.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+
+modularize()
+{
+    cp $1 modules/
+    cd modules
+    filename=$(basename "$1")
+    xxd -i "$filename" > ${filename}.c
```

Comment:
Dumping all the binary bits into a source file seems a little heavy handed, is it possible to use EFI_LOAD_FILE protocol instead?